README.md

```diff
@@ -31,31 +31,15 @@ The items on this checklist come from various sources, such as <a href="https://
 - [ ] We understand how to split the data into train/val/test to avoid data leakage
 - [ ] If we need to collect data, we know how difficult and costly it will be to collect and annotate
 - [ ] We have a plan for how to store and version our data, dataset splits, models, and change in annotations
-- [ ] We get a reasonable <a href="https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/aad9f93b86b7addfea4c419b9100c6cdd26cacea.pdf">“ML Test Score”</a>, table 1
 
 ## Modelling
 - [ ] We have one or several well thought out baselines in place. These are not good enough, so there’s an actual need to use ML
 - [ ] There’s a metrics webpage where we can compare runs and the url is ___________________
 - [ ] We can (approximately) reproduce a model if needed
-- [ ] We get a reasonable <a href="https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/aad9f93b86b7addfea4c419b9100c6cdd26cacea.pdf">“ML Test Score”</a>, table 2
 
 ## Deployment
-- [ ] We have CI in place
```

Comment:
Until we have a proper generalized, easy-to-use infra setup for onboarding new services, it could be nice to keep these around under a different header perhaps?

Reply:
I think the idea is that we kickstart that work and remove from here. Isn't that what we said on the kickoff day?

Reply:
Hmm, I am not sure we can kickstart work for all of these bullets, my thinking is that perhaps for some of these we would start by making it work for us, like with Github Actions. I think the w we still need a place where it says that we should use Github Actions because it is not something decided for all.